crates/evm/fuzz/src/strategies/param.rs

```diff
@@ -224,6 +229,169 @@ pub fn fuzz_param_from_state(
+                DynSolValue::Int(mutated_val, size)
+            }),
+            4 => I256::mutate_interesting_dword(val, size, test_runner).map(|mutated_val| {
+                trace!(target: "abi_mutation", "I256 interesting dword: {val} -> {mutated_val}");
+                DynSolValue::Int(mutated_val, size)
+            }),
+            5 => None,
+            _ => unreachable!(),
+        },
+        // Address: flip random bit or generate new value from state.
+        DynSolValue::Address(val) => match test_runner.rng().random_range(0..=1) {
+            0 => Address::flip_random_bit(val, None, test_runner).map(DynSolValue::Address),
+            1 => None,
+            _ => unreachable!(),
+        },
+        DynSolValue::Array(mut values) => {
+            if let DynSolType::Array(param_type) = param
+                && !values.is_empty()
+            {
+                match test_runner.rng().random_range(0..=2) {
+                    // Decrease array size by removing a random element.
+                    0 => {
+                        values.remove(test_runner.rng().random_range(0..values.len()));
+                    }
+                    // Increase array size.
+                    1 => values.push(new_value(param_type, test_runner)),
+                    // Mutate random array element.
+                    2 => mutate_array(&mut values, param_type, test_runner, state),
+                    _ => unreachable!(),
+                }
+                Some(DynSolValue::Array(values))
+            } else {
+                None
+            }
+        }
+        DynSolValue::FixedArray(mut values) => {
+            if let DynSolType::FixedArray(param_type, _size) = param
+                && !values.is_empty()
+            {
+                mutate_array(&mut values, param_type, test_runner, state);
+                Some(DynSolValue::FixedArray(values))
+            } else {
+                None
+            }
+        }
+        DynSolValue::CustomStruct { name, prop_names, tuple: mut values } => {
+            if let DynSolType::CustomStruct { name: _, prop_names: _, tuple: tuple_types }
+            | DynSolType::Tuple(tuple_types) = param
+                && !values.is_empty()
+            {
+                // Mutate random struct element.
+                mutate_tuple(&mut values, tuple_types, test_runner, state);
+                Some(DynSolValue::CustomStruct { name, prop_names, tuple: values })
+            } else {
+                None
+            }
+        }
+        DynSolValue::Tuple(mut values) => {
+            if let DynSolType::Tuple(tuple_types) = param
+                && !values.is_empty()
+            {
+                // Mutate random tuple element.
+                mutate_tuple(&mut values, tuple_types, test_runner, state);
+                Some(DynSolValue::Tuple(values))
+            } else {
+                None
+            }
+        }
+        _ => None,
```

Comment:
We can mutate FixedBytes variant too? It has a size argument and it could be treated the same as U256